### ISSUE TITLE
Kill package-lock.json

### ISSUE THREAD

**Issue by mokone91** [voidcosmos/npkill#117]
Thanks for this wonderful tool!

is it possible to add the ability to remove package-lock.json files too? 
Thanks!

**Comment by hd321kbps** [comment]
Convenient addition. Can be used via flag

**Comment by zaldih** [comment]
Thank you for your comment!
I must admit that I have been thinking about this issue for several days.

There is the `--target` parameter in which you can specify a different target than node_modules but it certainly won't work since for efficiency it only searches between directories... wait, Efficiency?

I have done some tests and at least in unix there is hardly any difference between filtering by files and directories, so it will be easy to ajust.

On the other hand for windows it will require some extra modification. I will work on it for the next version.

**TLDR:** Soon with the --target flag you will be able to specify also files instead of only directories so you will be able to use `npkill --target package-lock.json` .

**Comment by zaldih** [comment]
### Done at 6f8c1caf6a7662d1b4e16bc9fb17b9faa2feb126

In the next release it will also be possible to specify **files** as **target** as requested.
